draft-ietf-mls-protocol.md

```diff
@@ -2259,6 +2258,12 @@ struct {
 } Update;
 ~~~~~
 
+The KeyPackage contained in an `Update` proposal MUST share the values of the
```

Comment:
Rather than say "share" here, it would be clearer to say that the values MUST be equal.

Reply:
Likewise below, instead of "MUST NOT share", "MUST be different".

Reply:
Agreed